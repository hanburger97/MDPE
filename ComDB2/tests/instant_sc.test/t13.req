select i, j, k, v from t
select i, j, k, v from t order by i
select i, j, k, v from t order by j

insert into t(i, j) values(34, 4207848106)
insert into t(i, j, k, v) values(35, 4207861421, 35, 'thirty five')

select i, j, k, v from t
select i, j, k, v from t order by i
select i, j, k, v from t order by j
