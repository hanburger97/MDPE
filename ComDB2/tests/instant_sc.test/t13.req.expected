(i=0, j=0, k=99, v='comdb2')
(i=1, j=1, k=99, v='comdb2')
(i=2, j=2, k=99, v='comdb2')
(i=3, j=3, k=99, v='comdb2')
(i=4, j=4, k=99, v='comdb2')
(i=5, j=5, k=99, v='comdb2')
(i=6, j=6, k=99, v='comdb2')
(i=7, j=7, k=99, v='comdb2')
(i=8, j=8, k=99, v='comdb2')
(i=9, j=9, k=99, v='comdb2')
(i=10, j=10, k=99, v='comdb2')
(i=11, j=11, k=99, v='comdb2')
(i=12, j=12, k=99, v='comdb2')
(i=13, j=13, k=99, v='comdb2')
(i=14, j=14, k=99, v='comdb2')
(i=15, j=15, k=15, v='comdb2')
(i=16, j=16, k=16, v='comdb2')
(i=17, j=17, k=17, v='comdb2')
(i=18, j=18, k=18, v='comdb2')
(i=19, j=19, k=19, v='comdb2')
(i=20, j=20, k=999, v='comdb2')
(i=21, j=21, k=999, v='comdb2')
(i=22, j=22, k=999, v='comdb2')
(i=23, j=23, k=999, v='comdb2')
(i=24, j=24, k=999, v='comdb2')
(i=25, j=25, k=25, v='20+5')
(i=26, j=26, k=26, v='twenty six')
(i=27, j=27, k=27, v='twenty seven')
(i=28, j=28, k=28, v='twenty eight')
(i=29, j=29, k=29, v='twenty nine')
(i=30, j=3405691582, k=30, v='thirty')
(i=31, j=3735928559, k=999, v='comdb2')
(i=32, j=3735936685, k=999, v='comdb2')
(i=33, j=3735943886, k=33, v='thirty three')
(i=0, j=0, k=99, v='comdb2')
(i=1, j=1, k=99, v='comdb2')
(i=2, j=2, k=99, v='comdb2')
(i=3, j=3, k=99, v='comdb2')
(i=4, j=4, k=99, v='comdb2')
(i=5, j=5, k=99, v='comdb2')
(i=6, j=6, k=99, v='comdb2')
(i=7, j=7, k=99, v='comdb2')
(i=8, j=8, k=99, v='comdb2')
(i=9, j=9, k=99, v='comdb2')
(i=10, j=10, k=99, v='comdb2')
(i=11, j=11, k=99, v='comdb2')
(i=12, j=12, k=99, v='comdb2')
(i=13, j=13, k=99, v='comdb2')
(i=14, j=14, k=99, v='comdb2')
(i=15, j=15, k=15, v='comdb2')
(i=16, j=16, k=16, v='comdb2')
(i=17, j=17, k=17, v='comdb2')
(i=18, j=18, k=18, v='comdb2')
(i=19, j=19, k=19, v='comdb2')
(i=20, j=20, k=999, v='comdb2')
(i=21, j=21, k=999, v='comdb2')
(i=22, j=22, k=999, v='comdb2')
(i=23, j=23, k=999, v='comdb2')
(i=24, j=24, k=999, v='comdb2')
(i=25, j=25, k=25, v='20+5')
(i=26, j=26, k=26, v='twenty six')
(i=27, j=27, k=27, v='twenty seven')
(i=28, j=28, k=28, v='twenty eight')
(i=29, j=29, k=29, v='twenty nine')
(i=30, j=3405691582, k=30, v='thirty')
(i=31, j=3735928559, k=999, v='comdb2')
(i=32, j=3735936685, k=999, v='comdb2')
(i=33, j=3735943886, k=33, v='thirty three')
(i=0, j=0, k=99, v='comdb2')
(i=1, j=1, k=99, v='comdb2')
(i=2, j=2, k=99, v='comdb2')
(i=3, j=3, k=99, v='comdb2')
(i=4, j=4, k=99, v='comdb2')
(i=5, j=5, k=99, v='comdb2')
(i=6, j=6, k=99, v='comdb2')
(i=7, j=7, k=99, v='comdb2')
(i=8, j=8, k=99, v='comdb2')
(i=9, j=9, k=99, v='comdb2')
(i=10, j=10, k=99, v='comdb2')
(i=11, j=11, k=99, v='comdb2')
(i=12, j=12, k=99, v='comdb2')
(i=13, j=13, k=99, v='comdb2')
(i=14, j=14, k=99, v='comdb2')
(i=15, j=15, k=15, v='comdb2')
(i=16, j=16, k=16, v='comdb2')
(i=17, j=17, k=17, v='comdb2')
(i=18, j=18, k=18, v='comdb2')
(i=19, j=19, k=19, v='comdb2')
(i=20, j=20, k=999, v='comdb2')
(i=21, j=21, k=999, v='comdb2')
(i=22, j=22, k=999, v='comdb2')
(i=23, j=23, k=999, v='comdb2')
(i=24, j=24, k=999, v='comdb2')
(i=25, j=25, k=25, v='20+5')
(i=26, j=26, k=26, v='twenty six')
(i=27, j=27, k=27, v='twenty seven')
(i=28, j=28, k=28, v='twenty eight')
(i=29, j=29, k=29, v='twenty nine')
(i=30, j=3405691582, k=30, v='thirty')
(i=31, j=3735928559, k=999, v='comdb2')
(i=32, j=3735936685, k=999, v='comdb2')
(i=33, j=3735943886, k=33, v='thirty three')
(rows inserted=1)
(rows inserted=1)
(i=0, j=0, k=99, v='comdb2')
(i=1, j=1, k=99, v='comdb2')
(i=2, j=2, k=99, v='comdb2')
(i=3, j=3, k=99, v='comdb2')
(i=4, j=4, k=99, v='comdb2')
(i=5, j=5, k=99, v='comdb2')
(i=6, j=6, k=99, v='comdb2')
(i=7, j=7, k=99, v='comdb2')
(i=8, j=8, k=99, v='comdb2')
(i=9, j=9, k=99, v='comdb2')
(i=10, j=10, k=99, v='comdb2')
(i=11, j=11, k=99, v='comdb2')
(i=12, j=12, k=99, v='comdb2')
(i=13, j=13, k=99, v='comdb2')
(i=14, j=14, k=99, v='comdb2')
(i=15, j=15, k=15, v='comdb2')
(i=16, j=16, k=16, v='comdb2')
(i=17, j=17, k=17, v='comdb2')
(i=18, j=18, k=18, v='comdb2')
(i=19, j=19, k=19, v='comdb2')
(i=20, j=20, k=999, v='comdb2')
(i=21, j=21, k=999, v='comdb2')
(i=22, j=22, k=999, v='comdb2')
(i=23, j=23, k=999, v='comdb2')
(i=24, j=24, k=999, v='comdb2')
(i=25, j=25, k=25, v='20+5')
(i=26, j=26, k=26, v='twenty six')
(i=27, j=27, k=27, v='twenty seven')
(i=28, j=28, k=28, v='twenty eight')
(i=29, j=29, k=29, v='twenty nine')
(i=30, j=3405691582, k=30, v='thirty')
(i=31, j=3735928559, k=999, v='comdb2')
(i=32, j=3735936685, k=999, v='comdb2')
(i=33, j=3735943886, k=33, v='thirty three')
(i=34, j=4207848106, k=9999, v='comdb2')
(i=35, j=4207861421, k=35, v='thirty five')
(i=0, j=0, k=99, v='comdb2')
(i=1, j=1, k=99, v='comdb2')
(i=2, j=2, k=99, v='comdb2')
(i=3, j=3, k=99, v='comdb2')
(i=4, j=4, k=99, v='comdb2')
(i=5, j=5, k=99, v='comdb2')
(i=6, j=6, k=99, v='comdb2')
(i=7, j=7, k=99, v='comdb2')
(i=8, j=8, k=99, v='comdb2')
(i=9, j=9, k=99, v='comdb2')
(i=10, j=10, k=99, v='comdb2')
(i=11, j=11, k=99, v='comdb2')
(i=12, j=12, k=99, v='comdb2')
(i=13, j=13, k=99, v='comdb2')
(i=14, j=14, k=99, v='comdb2')
(i=15, j=15, k=15, v='comdb2')
(i=16, j=16, k=16, v='comdb2')
(i=17, j=17, k=17, v='comdb2')
(i=18, j=18, k=18, v='comdb2')
(i=19, j=19, k=19, v='comdb2')
(i=20, j=20, k=999, v='comdb2')
(i=21, j=21, k=999, v='comdb2')
(i=22, j=22, k=999, v='comdb2')
(i=23, j=23, k=999, v='comdb2')
(i=24, j=24, k=999, v='comdb2')
(i=25, j=25, k=25, v='20+5')
(i=26, j=26, k=26, v='twenty six')
(i=27, j=27, k=27, v='twenty seven')
(i=28, j=28, k=28, v='twenty eight')
(i=29, j=29, k=29, v='twenty nine')
(i=30, j=3405691582, k=30, v='thirty')
(i=31, j=3735928559, k=999, v='comdb2')
(i=32, j=3735936685, k=999, v='comdb2')
(i=33, j=3735943886, k=33, v='thirty three')
(i=34, j=4207848106, k=9999, v='comdb2')
(i=35, j=4207861421, k=35, v='thirty five')
(i=0, j=0, k=99, v='comdb2')
(i=1, j=1, k=99, v='comdb2')
(i=2, j=2, k=99, v='comdb2')
(i=3, j=3, k=99, v='comdb2')
(i=4, j=4, k=99, v='comdb2')
(i=5, j=5, k=99, v='comdb2')
(i=6, j=6, k=99, v='comdb2')
(i=7, j=7, k=99, v='comdb2')
(i=8, j=8, k=99, v='comdb2')
(i=9, j=9, k=99, v='comdb2')
(i=10, j=10, k=99, v='comdb2')
(i=11, j=11, k=99, v='comdb2')
(i=12, j=12, k=99, v='comdb2')
(i=13, j=13, k=99, v='comdb2')
(i=14, j=14, k=99, v='comdb2')
(i=15, j=15, k=15, v='comdb2')
(i=16, j=16, k=16, v='comdb2')
(i=17, j=17, k=17, v='comdb2')
(i=18, j=18, k=18, v='comdb2')
(i=19, j=19, k=19, v='comdb2')
(i=20, j=20, k=999, v='comdb2')
(i=21, j=21, k=999, v='comdb2')
(i=22, j=22, k=999, v='comdb2')
(i=23, j=23, k=999, v='comdb2')
(i=24, j=24, k=999, v='comdb2')
(i=25, j=25, k=25, v='20+5')
(i=26, j=26, k=26, v='twenty six')
(i=27, j=27, k=27, v='twenty seven')
(i=28, j=28, k=28, v='twenty eight')
(i=29, j=29, k=29, v='twenty nine')
(i=30, j=3405691582, k=30, v='thirty')
(i=31, j=3735928559, k=999, v='comdb2')
(i=32, j=3735936685, k=999, v='comdb2')
(i=33, j=3735943886, k=33, v='thirty three')
(i=34, j=4207848106, k=9999, v='comdb2')
(i=35, j=4207861421, k=35, v='thirty five')
